WCFDataService/Client/System/Data/Services/Client/ClientEdmModel.cs

```diff
@@ -495,12 +500,43 @@ private EdmTypeCacheValue GetOrCreateEdmTypeInternal(IEdmStructuredType edmBaseT
         }
 
         /// <summary>
+        /// Tries to find the backing field for property on type.
+        /// </summary>
+        /// <param name="property">The property to find the backing field for.</param>
+        /// <returns>The property backing field or null.</returns>
+        private static FieldInfo FindBackingField(PropertyInfo property, ProxyBackingFieldNamingConvention proxyBackingFieldNamingConvention)
```

Comment:
Clarification - I will ensure we only do this for generic collection of entites AND for complex properties.